.github/workflows/pull-request.yaml

```diff
@@ -12,6 +12,9 @@ jobs:
       CI_TOOLS_URL: https://github.com/samuelattwood/partner-charts-ci/releases/latest/download/partner-charts-ci-linux-amd64
 
     steps:
+      - uses: actions/setup-node@v2
```

Comment:
Why not the latest version?

Reply:
same as the above comment about the versions

Reply:
Node 16 is EOL. https://endoflife.date/nodejs

Reply:
I don't see the value of not using the latest version of the official Github Actions.